web-chat/src/ChatList.tsx

```diff
@@ -1,73 +1,38 @@
-import {
-  Card,
-  CardContent,
-  List,
-  ListItem,
-  ListItemText,
-  Typography,
-} from '@material-ui/core';
-import React, { useEffect, useRef } from 'react';
+import { useEffect, useRef } from 'react';
 import { ChatMessage } from '../../build/main/chat/chat_message';
 
+import { Message, MessageText, MessageGroup, Bubble } from '@livechat/ui-kit';
+
 interface Props {
   messages: ChatMessage[];
 }
 
 export default function ChatList(props: Props) {
   const messages = props.messages;
 
-  const listItems = messages.map((message) => (
-    <ListItem key={message.timestamp.toString()}>
-      <ListItemText primary={<Message message={message} />} />
-    </ListItem>
+  const listItems = messages.map((currentmessage) => (
+    <Message
+      key={currentmessage.timestamp.toString()}
+      authorName={currentmessage.nick}
+      date={currentmessage.timestamp.toLocaleString([], {
```

Comment:
Maybe best if we extract the formatting of the date in a dedicated function to declutter.

Reply:
Done (please review)